src/Components/NewBlog.js

```diff
@@ -20,7 +22,7 @@ const CreatePost = () => {
   const handleSubmit = async (e) => {
     e.preventDefault();
     try {
-      await axios.post("https://blogpostapi1.herokuapp.com/", {
+      await axios.post(postApiUrl, {
         name: post.name,
         title: post.title,
         content: post.content,
```

Comment:
You can pass in `post`, instead of `{name: post.name, title: post.title, content: post.content}`